tools/__init__.py

```diff
@@ -7,3 +10,12 @@ def split_email_address(email_address):
     if len(parts) != 2:
         raise ValueError('Not an email passed: %s' % email_address)
     return parts[0], parts[1]
+
+
+def get_file_extension(git_file_path, max_ext_length=5):
```

Comment:
I think 5 chars for the extension is a bit limited. One of the repos I run haikuports in has mainly "recipe" files so you can see that's a problem for me :)

I think it would make sense, if possible, to collect extensions without length limitation, but then collapse all the extensions with very few matching files in an "other" group. Also it would be great if this would handle special filenames like "Makefile", but I should probably open a ticket about that

Reply:
There is max_ext_length setting in configuration file and that one supposed to be used to control what we can consider as an extension. But looks like it never was used...

Please, create an Issue with description of what you would like to see, I'll schedule it.